packages/analytics/google/index.tsx

```diff
@@ -1,5 +1,7 @@
 import { AddToCartInput, CartLineItem } from '@ifixit/cart-sdk';
 import { moneyToNumber, parseItemcode } from '@ifixit/helpers';
+import { debounce } from 'lodash';
```

Comment:
There's an 8kb increment in the bundle size, and this might be the cause since it'll end up importing the whole lodash library

```suggestion
import debounce from 'lodash/debounce';
```